assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -245,3 +245,34 @@
 		}
 	}
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size {
+
+	.elementor-control-field {
+		padding: 0;
+	}
+
+	// Control won't support more than one breakpoints
```

Comment:
explain it more (about the bug and everything we've talked about)

Reply:
Done